### PR TITLE
Make `packages/desktop` compilable

### PR DESCRIPTION
**Pull request by retcinder**
It never actually worked for me so I made it work.

## TODO
* [x] System tray
  * [x] Make menu not appear on left click (instead, show window) (Unsupported on tauri's end for linux)

### CODE REVIEW

**Review by catuhana — COMMENTED** [review]
[Rust Style Guide recommends using spaces instead of tabs.](https://doc.rust-lang.org/nightly/style-guide/#indentation-and-line-width) Also this change only converts spaces to tabs on VSCode, [`rustfmt.toml` file should be used instead.](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#hard_tabs)

**Comment by retcinder** [comment]
> [Rust Style Guide recommends using spaces instead of tabs.](https://doc.rust-lang.org/nightly/style-guide/#indentation-and-line-width) Also this change only converts spaces to tabs on VSCode, [`rustfmt.toml` file should be used instead.](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#hard_tabs)

I disagree with the rust styleguide. Also, that change for vscode only was on purpose as vscode was overriding to the project default (2 wide spaces)

File: packages/desktop/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -1,27 +1,25 @@
 [package]
-name = "upryzing"
-version = "0.1.0"
-description = "Your conversations, your way."
-authors = ["Vexcited"]
-license = "MIT"
+name = "Upryzing"
+version = "0.0.1"
+description = "Your Conversations, Your Way"
```
Comment (Rexogamer) [inline packages/desktop/src-tauri/Cargo.toml]: why was this changed?

File: packages/desktop/src-tauri/src/menu.rs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+	let a = AboutMetadata {
+		name: Some("Upryzing".to_string()),
+		version: Some("0.0.1".to_string()),
+		short_version: Some("0.0.1".to_string()),
+		authors: Some(authors.to_vec()),
+		comments: Some("".to_string()),
+		copyright: Some("".to_string()),
+		license: Some("AGPL-v3".to_string()),
+		website: Some("".to_string()),
+		website_label: Some("".to_string()),
```
Comment (Rexogamer) [inline packages/desktop/src-tauri/src/menu.rs]: this could be filled in

**Review by Rexogamer — DISMISSED** [review]

File: packages/desktop/src-tauri/Cargo.toml (hunk 1 above)
Comment (retcinder) [reply packages/desktop/src-tauri/Cargo.toml]: The entire tauri system was essentially replaced

Comment (Rexogamer) [reply packages/desktop/src-tauri/Cargo.toml]: I meant the capitalisation

File: packages/desktop/src-tauri/src/menu.rs (hunk 2 above)
Comment (retcinder) [reply packages/desktop/src-tauri/src/menu.rs]: What would you say should be put on website_label?
(It acts as the text for a hyperlink)

@Rexogamer

File: packages/desktop/src-tauri/Cargo.toml (hunk 1 above)
Comment (retcinder) [reply packages/desktop/src-tauri/Cargo.toml]: Oh, whoops. Changing it back.

File: packages/desktop/src-tauri/src/menu.rs (hunk 2 above)
Comment (retcinder) [reply packages/desktop/src-tauri/src/menu.rs]: Nevermind.

**Review by Rexogamer — APPROVED** [review]
